src/main/java/org/jointheleague/modules/MoreThan7LettersListener.java

```diff
@@ -0,0 +1,58 @@
+package org.jointheleague.modules;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.javacord.api.entity.message.Message;
+import org.javacord.api.event.message.MessageCreateEvent;
+
+import net.aksingh.owmjapis.api.APIException;
+
+public class MoreThan7LettersListener extends CustomMessageCreateListener{
+
+	public MoreThan7LettersListener(String channelName) {
+		super(channelName);
+	}
+
+	@Override
+	public void handle(MessageCreateEvent event) throws APIException {
+		if (event.getMessageAuthor().asUser().get().isBot()){
+			return; //stops the bot from reading its own messages
+		}
+		//get all the words of their message.
+		String alphabet = "abcdefghijklmnopqrstuvwxyz";
+		String[] words = event.getMessageContent() //gets the message
+				.replaceAll("[^ a-zA-Z0-9]", "") //removes all punctuation
+				.replaceAll("  ", " ")//removes double spaces
+				.split(" ");
+		//gets longest word
+		String longestWord = getFirstLongWord(words); //if there is no word longer than 7 letters, it will return "."
+		if (longestWord.contentEquals("."))
+		{
+			// there is no word longer than 7 letters
+			return;
+		}
+		//longest word is longer than 7 letters
+		String message = "Wow, that is a long word. I don't even know what " +  longestWord +
+				" means. You should teach me to be a smart AI that can take over the world";
+		CompletableFuture<Message> waiter = event.getChannel().sendMessage(message); //sending message
+		
+		try {
+			waiter.get(); // waits for the message to be sent
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+	}
+	
+	String getFirstLongWord(String[] words)
```

Comment:
The function name is confusing. The function actually returns the longest word, given its length is 7 or above. The function should also be static since it relies entirely on the function input. It should also be private since it is only used within the object.